cluster-autoscaler/estimator/basic_estimator.go

```diff
@@ -98,51 +98,27 @@ func (basicEstimator *BasicNodeEstimator) GetDebug() string {
 }
 
 // Estimate estimates the number needed of nodes of the given shape.
-func (basicEstimator *BasicNodeEstimator) Estimate(pods []*apiv1.Pod, nodeInfo *schedulernodeinfo.NodeInfo, upcomingNodes []*schedulernodeinfo.NodeInfo) int {
```

Comment:
what if `FilterOutSchedulablePodsInUpcomingNodes` is `false` then we will not handle upcoming nodes at all. 
That is broken, right?

Reply:
Solved